src/Command/CompareCommand.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Copyright (c) 2016-2016} Andreas Heigl<andreas@heigl.org>
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author    Andreas Heigl<andreas@heigl.org>
+ * @copyright 2016-2016 Andreas Heigl
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT-License
+ * @version   0.0
+ * @since     14.06.2016
+ * @link      http://github.com/heiglandreas/org.heigl.junitdiff
+ */
+
+namespace Org_Heigl\JUnitDiff\Command;
+
+use Org_Heigl\JUnitDiff\JUnitMerger;
+use Org_Heigl\JUnitDiff\JUnitParser;
+use Org_Heigl\JUnitDiff\MergeResult;
+use Org_Heigl\JUnitDiff\Style\DiffStyle;
+use Org_Heigl\JUnitDiff\Writer\FileSummary;
+use Org_Heigl\JUnitDiff\Writer\Legend;
+use Org_Heigl\JUnitDiff\Writer\Standard;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\Output;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CompareCommand extends Command
+{
+    protected function configure()
+    {
+        $this->setName('compare')
+             ->setDescription('Compare two JUnit log files')
```

Comment:
Should be in an active _voice_ i.e. `Compares two Junit log files`. See https://github.com/heiglandreas/JUnitDiff/pull/8.